string-constants-lib/string-constants/private/danish-string-constants.rkt

```diff
@@ -1243,6 +1291,175 @@ please adhere to these guidelines:
+  (pkg-manager-menu-item "Pakkehåndtering…")
+  ;; where ~a gets an installation name:
+  (install-pkg-packages-for "Pakker for ~a")
+  (install-pkg-really-remove-installation "Er du sikker på, at du vil fjerne alle installerede pakker og information for ~a?")
+  
+  (install-pkg-abort-set-scope "Afbryd ændring af anvendelsesområde")
+
+  (install-pkg-dependencies-fail "Fejle: afbryder installering/opdatering hvis afhængigheder mangles")
+  (install-pkg-dependencies-force "Tving: installér på trods af mangler eller versionsmisforhold ved afhængigheder")
+  (install-pkg-dependencies-search-ask "Spørg: spørg ved hvért manglende afhængighed (virker ikke i GUI)")
+  (install-pkg-dependencies-search-auto "Auto: installér automatisk afhængigheder som mangler eller har forkert version")
+  (install-pkg-dependencies-search-auto+update "Auto + Opdater: Opdater afhængigheder når det er muligt")
+
+  (install-pkg-dependencies-mode "Afhængighedsindstilling")
+
+  (install-pkg-dependencies-search-ask-not-supported-in-gui
+   "“Spørg“ ved afhængigheder understøttes ikke i GUI.")
+  ;; "~a" is pre-":" part of `install-pkg-dependencies-fail' or `install-pkg-dependencies-search-auto':
+  (install-pkg-deps-is "Standard for afhængighedshåndtering er ~a")
+
+  (install-pkg-package-catalogs "Pakkekataloger") ; label for a list box
+  (install-pkg-add-package-catalog "Tilføj pakkekatalog")
+
+  (install-pkg-not-rentrant "Installering og opdatering kan ikke afvikles samtidigt;"
+                            " enten abryd den ene eller vent på at den bliver færdig.")
+  
+  ;; open a file via a collection path (new "Open" menu item in DrRacket)
+  (open-require-path "Åbn Require-sti…")
+  (enter-subcollection "Gå ind i subcollection") ; button in new dialog
+  (path-to-racket-binary "Sti til programmet")
+  (use-a-different-racket "Brug en anden Racket")
+
+  ;; adding racket/bin to the path; only under mac os x
+  ; first ~a is filled with either the empty string or an error message from elsewhere
+  ;  (bracketed by some newlines to separate it out)
+  ; second ~a is filled with /etc/paths.d/racket (or some other path like it in the future)
+  ; third ~a is filled with the path to the bin directory of the current drracket
+  (adding-racket/bin-to-path-failed 
+   "Fejl under forsøg ved at opsætte Racket til kommandolinjen. ~a"
+   " kunne ikke skabe “~a” med indhold “~a”.")
+  ; first and third ~a are filled with /etc/paths.d/racket (or some other path like it in the future)
+  ; and the second one is filled with the path to the bin directory that was put into that file.
+  (added-racket/bin-to-path
+   "Du bør nu kunne bruge racket og de tilknyttede værktøjer ved"
+   " kommandolinjen.\n\n"
+   " Standardstien PATH blev instillet for alle brugere ved at ændre filen"
+   " ~a til at pege på ~a. Du kan trække denne indstilling tilbage ved at slette"
+   " ~a.")
+  (add-racket/bin-to-path "Indstil kommandolinje for Racket…") ;; menu item label
   )
```

Comment:
Godt arbejde! (Very nice work! )

Btw - this dictionary is pretty good for words related to information technology.

http://www.klid.dk/dansk/ordlister/ordliste.html

Reply:
The difficulty with Danish translations of software is that it's hard to know which sets of words to use. E.g. Mac OS calls a menu "Arkiv", others call it "Fil" or "Filer". 

That list is good, though --- a lot of Danish IT wordlists feel like they're from either DIKU or the 70s, with "fastpladelager", "løsen", og "programmel" instead of the now-standard English loanwords. But this one seems to actually match the way the people who I know speak!

Thanks for the review!